libs/auth-api-lib/src/lib/delegations/delegations-incoming.service.ts

```diff
@@ -101,10 +101,14 @@ export class DelegationsIncomingService {
   async findAllAvailable(
     user: User,
     delegationTypes?: DelegationType[],
+    requestedScopes?: string[],
```

Comment:
~When working with few optional parameters it can be nice to switch to object, so when the function is called you don't have to pass undefined as a parameter, i.e.:
But it requires updating the code where the delegationTypes are used but it should be just one or two places.

```
interface FindAvailableInput {
  user: User
  delegationTypes?: DelegationType[]
  requestedScopes?: string[]
}

async findAvailable({
  user.
  delegationTypes
  requestedScopes
}: FindAvailableInput): Promise<MergedDelegationDTO[]> {
  ....
}

// Then call it in the controller without delegationTypes.
findAllAvailable({user, requestedScopes})
```

Reply:
Fixed.